integration-tests/src/tests/client/flat_storage.rs

```diff
@@ -432,31 +433,25 @@ fn test_flat_storage_iter() {
 
         match shard_id {
             0 => {
-                // Two entries - one for account, the other for contract.
                 assert_eq!(2, items.len());
+                // Two entries - one for 'near' system account, the other for the contract.
```

Comment:
Here I'm guessing we are just shuffling accounts around based on shard layout? I can see "near" has moved from shard 2 to shard 0.

Reply:
yep, previously test0 and test1 were fixed, now they are boundaries and it moved things around